app/workers/vic/delete_old_uploads.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module VIC
+  class DeleteOldUploads
+    include Sidekiq::Worker
+
+    def perform
```

Comment:
can you add `VIC::TagSentry.tag_sentry` here?

Reply:
I added this line below, it just didn't mark the comment as being taken care of. This should be ready for review now.